src/Voron/Impl/Paging/CryptoPager.cs

```diff
@@ -70,8 +70,8 @@ public unsafe class EncryptionBuffer
         public static readonly UIntPtr HashSize = Sodium.crypto_generichash_bytes();
         public static readonly int HashSizeInt = (int)Sodium.crypto_generichash_bytes();
         public byte* Pointer;
-        public int Size;
-        public int? OriginalSize;
+        public long Size;
```

Comment:
When dealing with large data (like attachments) we can have single allocation which are more than 2GB. So we ran into `int` overflow issue, right?

Reply:
yes